client/wp-admin-scripts/wc-pay-payments-promotion/index.tsx

```diff
@@ -15,10 +15,15 @@ const container = document.querySelector(
 if ( container ) {
 	const sortColumn = container.children[ 0 ].innerHTML;
 	const descriptionColumn = container.children[ 3 ].innerHTML;
+	const subTitle = container.getElementsByClassName( 'gateway-subtitle' );
+
 	render(
 		<WCPaymentsRow
```

Comment:
Is the idea that we may add more payment promotions in the future?  If so, this component name might be better if it's a bit more generic.